pom.xml

```diff
@@ -70,15 +70,15 @@
   </profiles>
 
   <properties>
-    <jackson.version>2.9.8</jackson.version>
+    <jackson.version>2.9.9</jackson.version>
```

Comment:
If you rebase your branch, this should already be bumped to `2.9.9`.

Reply:
done